### ISSUE TITLE
Cannot run windbg extension commands on Windows

### ISSUE THREAD

**Issue by xusheng6** [Vector35/debugger#152]
```
(Local DBGENG) !peb
SECURE: File not allowed to be loaded - C:\Windows\SYSTEM32\dbghelp.dll
Error code: Win32 error 0n5
The call to LoadLibrary(ext) failed, Win32 error 0n2
    "The system cannot find the file specified."
Please check your debugger configuration and/or network access.
No export peb found
```

The failure is not surprising since we fail to load the module. So the problem reduces to why it happens. This affects Win10/11 in the same way.

**Comment by xusheng6** [comment]
related to #150

**Comment by xusheng6** [comment]
Fixed in BN 3.1.3529-dev
